database/sqlite/sqlite_aclk_alert.c

```diff
@@ -1109,6 +1109,7 @@ void aclk_push_alarm_checkpoint(RRDHOST *host __maybe_unused)
             else if (active_alerts[i].status == RRDCALC_STATUS_CRITICAL)
                 buffer_strcat(alarms_to_hash, "C");
         }
+        freez(active_alerts);
```

Comment:
Hello @stelfrag ,

While I was running netdata compiled against `libasan` I had the impression the issue was fixed, but when I ran netdata with `valgrind` I still receive the report:

```sh
==22001== 152 bytes in 1 blocks are still reachable in loss record 4,530 of 5,818
==22001==    at 0x48475FF: calloc (vg_replace_malloc.c:1328)
==22001==    by 0x4521E8: malloc_trace_find_or_create (libnetdata.c:269)
==22001==    by 0x45288A: callocz_int (libnetdata.c:324)
==22001==    by 0x4F637A: aclk_push_alarm_checkpoint (sqlite_aclk_alert.c:1078)
==22001==    by 0x4A3BB3: health_main (health.c:1501)
==22001==    by 0x464C19: netdata_thread_init (threads.c:262)
==22001==    by 0x5A86CA9: start_thread (in /lib64/libc-2.37.so)
==22001==    by 0x5B10CBF: clone (in /lib64/libc-2.37.so)
```

Reply:
Thanks @thiagoftsm , I will take a look!